weldx/schemas/weldx.bam.de/weldx/core/media_file-0.1.0.yaml

```
@@ -29,17 +31,18 @@ properties:
     description: |
       In case of time-dynamic media, the time resolution (frames per second).
     tag: "asdf://weldx.bam.de/weldx/tags/units/quantity-0.1.0"
+    wx_unit: "1/s"
   resolution:
     description: |
       Media resolution in pixels.
     type: array
+    wx_shape: [n. m]
```

is this the same as `[n, m]`? (I am not sure how the YAML parsing works in this case)

if this is to store the resolution as a shape-like list of two integers the following is works better:
```suggestion
    type: array
    items:
      type: integer
      number: 2
```

(looking for the asdf/json array syntax I stumbled upon [this example](https://asdf.readthedocs.io/en/2.11.1/asdf/features.html#custom-schemas) in the ASDF docs that is probably pretty close to what we also want 😄 maybe look into that as well for reference 👍)

I *think* a `wx_shape` implementation would be more in line with
```suggestion
    type: array
    wx_shape: [1, 1]
```